README.md

```diff
@@ -5,3 +5,21 @@ A dashboard for the Helium [miner\_exporter](https://github.com/tedder/miner_exp
 # Installing
 
 Import (or copy/paste) the JSON, or **import by ID** `14319`, which uses the [grafana dashboard registry entry](https://grafana.com/grafana/dashboards/14319).
+
+# Prometheus labels per Validator
+This dashboard uses a $validator variable which assumes Prometheus metrics have a a label called "validator". The values of this label should be the short animal name of each validator. Example:
+```yaml
+...
+  - job_name: 'validator'
+    static_configs:
+            - targets: ['aa.yy.zzz:1234', 'aa.yy.zzz:1235']
+              labels:
+                      validator: 'angry-purple-tiger'
```

Comment:
I'm trying to think of a way we can self-discover these labels to cut down on the install effort. The validator name is already being sent via the exporter. We could send it much more explicitly. Thoughts?

Reply:
Yes, agree. I'll work on this. Also, after looking at many node_exporter graphs, it feels like they're all using a similar notion of "node" so let me spend some time figuring out what is commonly done and how to get it automagically.

Reply:
feel free to HMU on discord some more. I'm pretty new at prom/graf, unfortunately.